redfish-core/src/error_messages.cpp

```diff
@@ -1857,21 +1857,13 @@ nlohmann::json invalidUpload(std::string_view arg1, std::string_view arg2)
  */
 nlohmann::json restrictedRole(const std::string& arg1)
 {
-    return nlohmann::json{
-        {"@odata.type", "#Message.v1_1_1.Message"},
-        {"MessageId", "Base.1.9.0.RestrictedRole"},
-        {"Message", "The operation was not successful because the role '" +
-                        arg1 + "' is restricted."},
-        {"MessageArgs", {arg1}},
-        {"MessageSeverity", "Warning"},
-        {"Resolution",
-         "No resolution is required.  For standard roles, consider using the role "
-         "specified in the AlternateRoleId property in the Role resource."}};
+    return getLog(redfish::registries::base::Index::restrictedRole,
```

Comment:
This works the same? 
Tested this?

Reply:
Yes, I verified it.
It is how it is done in upstream instead of hard-corded (i.e. MSG version is older than later schema).